src/Altinn.Common/Altinn.Common.AccessToken/Altinn.Common.AccessToken/Altinn.Common.AccessToken.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <OutputType>Library</OutputType>
+    <Version>0.5.0</Version>
+    <Description>Package to verify Access Tokens from client. Require public certificates stored in Azure KeyVault</Description>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageProjectUrl></PackageProjectUrl>
+    <RepositoryType>https://github.com/Altinn/altinn-studio</RepositoryType>
+    <PackageTags>Altinn AccessToken</PackageTags>
+    <Company>Altinn</Company>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.5" />
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.4.0" />
```

Comment:
Consider adding StyleCop.Analysers and the rules we have in other projects like Storage.

Reply:
Done